### ISSUE TITLE
Adding a shared folder results in "Curl failed with error code: 22"

### ISSUE THREAD

**Issue by FlexServ** [OpenMediaVault-Plugin-Developers/openmediavault-btsync#4]
openmediavault 1.0.23
openmediavault-btsync 1.0.10

Trying to add a shard folder results in an error msg "Curl failed with error code: 22"

Details of the error msg:
```
Error #0:
exception 'Exception' with message 'Curl failed with error code: 22' in /usr/share/php/btsync/Api.php:188
Stack trace:
#0 /usr/share/php/btsync/Api.php(69): Btsync\Api->makeCall('add_folder', Array)
#1 /usr/share/openmediavault/engined/rpc/btsync.inc(453): Btsync\Api->addFolder('/media/26c0663a...', 'ASTALWZZ2DHOXX5...')
#2 [internal function]: OMVRpcServiceBtsync->set(Array, Array)
#3 /usr/share/php/openmediavault/rpcservice.inc(125): call_user_func_array(Array, Array)
#4 /usr/share/php/openmediavault/rpc.inc(79): OMVRpcServiceAbstract->callMethod('set', Array, Array)
#5 /usr/sbin/omv-engined(501): OMVRpc::exec('Btsync', 'set', Array, Array, 1)
#6 {main}
```

**Comment by nohatssir** [comment]
Closing this since it's an old issue now without any way to replicate it myself.

**Comment by nomadturk** [comment]
I did encounter the same error and fixed it.

Just for those who can't figure it out, these are the steps I did:

To resolve it,

First I had to open the GUI, create a user and accept the licence.
Secondly I checked the "Shared Folders" and edited the ACL to give recursive read/write permissions to www-data user/group and btsync user/groups.

Then applied settings.
From then on there was no error messages and BTSync
